oktoberfest/data/spectra.py

```diff
@@ -260,7 +266,7 @@ def get_matrix(self, fragment_type: FragmentType) -> Tuple[csr_matrix, List[str]
         layer = self._resolve_layer_name(fragment_type)
         matrix = self.layers[layer]
 
-        return matrix, self._gen_column_names(fragment_type)
+        return matrix, self._gen_column_names(fragment_type)  # , set(self.obs["FRAGMENTATION"]))
```

Comment:
here an also further down, you may want to check is you can remove the comments with the `set`